examples/with-apollo-and-redux/lib/initApollo.js

```diff
@@ -0,0 +1,36 @@
+import { ApolloClient, createNetworkInterface } from 'react-apollo'
+import fetch from 'isomorphic-fetch'
+
+let apolloClient = null
+
+// Polyfill fetch() on the server (used by apollo-client)
+if (!process.browser) {
+  global.fetch = fetch
+}
+
+function create () {
+  return new ApolloClient({
+    ssrMode: !process.browser, // Disables forceFetch on the server (so queries are only run once)
+    networkInterface: createNetworkInterface({
+      uri: 'https://api.graph.cool/simple/v1/cixmkt2ul01q00122mksg82pn', // Server URL (must be absolute)
+      opts: { // Additional fetch() options like `credentials` or `headers`
+        credentials: 'same-origin'
+      }
+    })
+  })
+}
+
+export default function initApollo () {
+  // Make sure to create a new client for every server-side request so that data
+  // isn't shared between connections (which would be bad)
```

Comment:
Server-side, this method is AFAIK called twice per server render (because of getDataFromTree + normal render). Would be nifty to remember the client within the same session so that `create()` gets called a single time (don't know if it's worth it though).

Similar comment for Redux store creation I guess.